scipy/stats/stats.py

```diff
@@ -5652,20 +5652,17 @@ def ttest_1samp(a, popmean, axis=0, nan_policy='propagate',
 
 def _ttest_finish(df, t, alternative):
     """Common code between all 3 t-test functions."""
-    # We use ``betainc`` here as it handles the case when ``nan``
```

Comment:
Oh, nice! I didn't know that `stdtr` handles masked arrays. This makes this function much simpler now. Thanks!